home-assistant-plugin/custom_components/xknx/switch.py

```diff
@@ -23,6 +23,27 @@ def __init__(self, device: XknxSwitch):
         """Initialize of KNX switch."""
         super().__init__(device)
 
+    @property
+    def current_power_w(self):
+        """Return current power consumption in W."""
+        return (
+            self._device.current_power
+            if self._device.current_power is not None
+            else None
+        )
```

Comment:
```suggestion
        return self._device.current_power
```
When self._device.current_power was None this would look like
`None if None is not None else None`

Reply:
Added in latest commit.